x/cronos/keeper/evm_log_handlers.go

```diff
@@ -136,7 +157,7 @@ func (h EthereumTransferHandler) Handle(ctx sdk.Context, contract common.Address
 	}
 
 	denom, found := h.cronosKeeper.GetDenomByContract(ctx, contract)
-	if !found {
+	if !found && !types.IsValidGravityDenom(denom) {
```

Comment:
```suggestion
	if !found || !types.IsValidGravityDenom(denom) {
```

I guess it should be a `||` here?

Reply:
fixed